code/LeftAndMain.php

```diff
@@ -297,6 +297,9 @@ class LeftAndMain extends Controller implements PermissionProvider
      */
     protected $versionProvider;
 
+    // phpcs:ignore
```

Comment:
```suggestion
    // phpcs:ignore PSR2.Classes.PropertyDeclaration.Underscore
```
Only ignore that specific rule.